bucket/dd.json

```diff
@@ -0,0 +1,9 @@
+{
+    "homepage": "http://www.chrysocome.net/ddhttp://www.chrysocome.net/dd",
```

Comment:
Typo in link? Should read just [http://www.chrysocome.net/dd](http://www.chrysocome.net/dd)?